src/main/java/org/folio/service/orders/OrderRolloverService.java

```diff
@@ -196,6 +217,32 @@ private Future<Void> rolloverOrdersByFundIds(List<String> chunkFundIds, LedgerFi
       .mapEmpty();
   }
 
+  private Future<Void> saveOrderLines(List<PoLine> orderLines, LedgerFiscalYearRollover ledgerFYRollover, PurchaseOrder.WorkflowStatus workflowStatus, RequestContext requestContext) {
+    return GenericCompositeFuture.join(orderLines.stream()
+        .map(poLine -> purchaseOrderLineService.saveOrderLine(poLine, requestContext)
+          .recover(t -> {
+            logger.error("PO line {} update failed while making rollover", poLine.getId(), t);
```

Comment:
better to extract code block in recover to separate method like 'handleRolloverFailure':
return GenericCompositeFuture.join(orderLines.stream()
        .map(poLine -> {
            return purchaseOrderLineService.saveOrderLine(poLine, requestContext)
                .recover(t -> handleRolloverFailure(poLine, ledgerFYRollover, workflowStatus, t, requestContext)
                    .map(a -> {
                        throw new HttpException(400, ErrorCodes.PO_LINE_ROLLOVER_FAILED.toError());
                    }));
        })
        .collect(toList()))
        .mapEmpty();